src/components/pipelines-overview/PipelineRunsStatusCard.tsx

```diff
@@ -59,10 +68,45 @@ const PipelinesRunsStatusCard: React.FC<PipelinesRunsStatusCardProps> = ({
     x: domainX || [startDate, endDate],
     y: domainY || undefined,
   };
+  const date = getDropDownDate(timespan).toISOString();
+
+  React.useEffect(() => {
+    const summaryOpt = {
+      summary: 'succeeded,cancelled,failed,others,total',
+      filter: `data.status.startTime>timestamp("${date}")`,
+    };
+    getResultsSummary(namespace, summaryOpt, DataType.PipelineRun)
+      .then((d) => setData(d))
+      .catch((e) => {
+        throw e;
+      });
+  }, [namespace, date]);
+
+  React.useEffect(() => {
+    const summaryOpt2 = {
+      summary: 'succeeded,cancelled,failed,others,total',
+      filter: `data.status.startTime>timestamp("${date}")`,
+      groupBy: 'day',
+    };
+    getResultsSummary(namespace, summaryOpt2, DataType.PipelineRun)
+      .then((d) => setData2(d))
+      .catch((e) => {
+        throw e;
+      });
+  }, [namespace, date]);
+
   const tickValues = getXaxisValues(timespan);
 
-  const chartData = tickValues.map((value) => {
-    return { x: value, y: 100 };
+  const chartData = tickValues?.map((value) => {
```

Comment:
The tooltip in the chart just shows the number like `24` , it is better to show `24%`?